ArduPlane/commands_logic.cpp

```diff
@@ -80,7 +80,7 @@ bool Plane::start_command(const AP_Mission::Mission_Command& cmd)
 
     case MAV_CMD_NAV_RETURN_TO_LAUNCH:
         set_mode(RTL, MODE_REASON_UNKNOWN);
-        break;
+    break;
```

Comment:
Kono break no indento ga hoka to kotonatte imasu. Watashi wa, hoka to awase howaitosupēsu 4 karamu o konomimasu.
50/5000
The indent of this break is different from the others. I like the white space 4 column as well as others.